### DISCUSSION TITLE
jcdickinson/codeium.nvim

### DISCUSSION THREAD

**Discussion by chenhuiapp** [Exafunction/codeium.nvim#102] [Q&A] [answered]
Hi, I am using the following to include codeium.nvim
```
  {
    "jcdickinson/codeium.nvim",
    dependencies = {
      -- "jcdickinson/http.nvim",
      "nvim-lua/plenary.nvim",
      "hrsh7th/nvim-cmp",
    },
    config = function() require("codeium").setup {} end,
  },
```
But it seems jcdickinson/codeium.nvim repo and Exfunction/codeium.vim are one repo now. Am I correct? Should I continue to use the above configuration or should I change to this repo? 

Thank you!

**Comment by pqn** [comment, marked as answer]
We adopted `jcdickinson/codeium.nvim`, and now it is `Exafunction/codeium.nvim`, which you can change to. `Exafunction/codeium.vim` remains separate (uses ghost text instead of nvim-cmp).

**Reply by chenhuiapp** [reply]
Thank you! Is there any difference between `Exafunction/codeium.nvim` and `Exafunction/codeium.vim`?

**Reply by pqn** [reply]
`codeium.nvim`/nvim-cmp will give you a different UI and integration with other completion providers, and `codeium.vim` gives you ghost text resembling Codeium on other IDEs, plus is the only one that works on vim.

**Reply by chenhuiapp** [reply]
Thank you!
